src/ViewHelpers/IfViewHelper.php

```diff
@@ -105,6 +119,29 @@
  *     The value of the "then" attribute is displayed if the condition evaluates to TRUE.
  *     Otherwise, everything the value of the "else"-attribute is displayed.
  *
+ * Combining multiple conditions
+ * -----------------------------
+ *
+ * ::
+ *
+ *     <f:if condition="{user.rank} > 100 && {user.type} == 'contributor'">
+ *         <f:then>
+ *             This is being shown in case both conditions matches.
+ *         </f:then>
+ *         <f:else if="{user.rank} > 200 && ({user.type} == 'contributor' || {user.type} == 'developer')">
+ *             This is being displayed in case the first condition evaluates to TRUE and either of the second
```

Comment:
"first condition" is ambigious: First I thought of the "user.rank > 100 && ..." condition